### PR TITLE
Node wrapping of core credentials API change

### PR DESCRIPTION
**Pull request by murgatroid99**

### CODE REVIEW

File: src/node/test/credentials_test.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ var fakeSuccessfulGoogleCredentials = {
 var fakeFailingGoogleCredentials = {
   getRequestMetadata: function(service_url, callback) {
     setTimeout(function() {
-      callback(new Error("Authorization failure"));
+      callback(new Error('Authorization failure'));
```
Comment (jboeuf) [inline src/node/test/credentials_test.js]: Authentication would be more correct here.

**Comment by jboeuf** [comment]
One very small nit but LGTM otherwise. Thanks much!

File: src/node/test/credentials_test.js (hunk 1 above)
Comment (murgatroid99) [inline src/node/test/credentials_test.js]: Fixed

**Comment by jboeuf** [comment]
LGTM. Thanks!

**Pull request merged**